django/db/backends/base/operations.py

```diff
@@ -799,6 +799,11 @@ def explain_query_prefix(self, format=None, **options):
     def insert_statement(self, on_conflict=None):
         return "INSERT INTO"
 
+    def generate_sql_for_conflict_target_field(self, name):
```

Comment:
This hook is unnecessary.

Reply:
are you sure? i thought everything backend-specific had to be managed at the backend-specific operation level. when we want to extend it to the other backend it will come in handy. but if you insist i'll move it.

Reply:
> This hook is unnecessary.

are you sure? i thought everything backend-specific had to be managed at the backend-specific operation level. when we want to extend it to the other backend it will come in handy. but if you insist i'll move it.

Reply:
This is not backend-specific. It looks the same on all backends that support the feature.

Reply:
> This is not backend-specific. It looks the same on all backends that support the feature.

for now. but when we want to hear it on mysql or mariaDB, we'll talk about VALUES or NEW depending on the mysql or mariaDB version. that's why i think it's better to leave it here. But it's up to you